### PR TITLE
Add breakpoints descriptions

### PR DESCRIPTION
**Pull request by TrySound**
Added tooltips to breakpoints similar to webflow.

image

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Comment by taylornowotny** [comment]
Updated tooltips @TrySound :

- Base - Styles on Base apply to all viewport sizes unless overwritten by another breakpoint. Start your styling here.
- 991 - Styles on this breakpoint apply to viewport widths 991px and down, unless overwritten by a smaller breakpoint.
- 767 - Styles on this breakpoint apply to viewport widths 767px and down, unless overwritten by a smaller breakpoint.
- 479 - Styles on this breakpoint apply to widths 479px and down.

**Comment by taylornowotny** [comment]
Updated tooltips @TrySound @kof :

I noticed that a tooltip was generated for custom breakpoints and I thought that was cool so here's a tooltip pattern that works for all breakpoints, custom or not:

**Base:** Styles on Base apply to all viewport sizes unless overwritten by another breakpoint. Start your styling here.
    
**Max-Width BP:** Styles on this breakpoint apply to viewport widths [value]px and down, unless overwritten by a smaller breakpoint.
    - EXAMPLE: 991: Styles on this breakpoint apply to viewport widths 991px and down, unless overwritten by a smaller breakpoint.

**Min-Width BP:** Styles on this breakpoint apply to viewport widths [value]px and up, unless overwritten by a larger breakpoint.

**Review by kof — APPROVED** [review]
@TrySound please update wording with @taylornowotny's suggestions and we are good to go

File: apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx [hunk 1]
```diff
@@ -8,9 +9,45 @@ import { CascadeIndicator } from "./cascade-indicator";
...
 import { selectedBreakpointIdStore } from "~/shared/nano-states";
-import { groupBreakpoints } from "~/shared/breakpoints";
+import { groupBreakpoints, isBaseBreakpoint } from "~/shared/breakpoints";
 import type { Breakpoint, Breakpoints } from "@webstudio-is/project-build";
 
+const getTooltipContent = (breakpoint: Breakpoint) => {
+  if (isBaseBreakpoint(breakpoint)) {
+    return (
+      <>
+        <strong>Base breakpoint</strong>
```
Comment (kof) [inline apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx:19]: could use <Text component it has a bold option

**Comment by TrySound** [comment]
This good?
image

**Comment by TrySound** [comment]
Or did you specifically mean "991: description" and not as heading?

**Comment by kof** [comment]
@TrySound I think the title like on webflow is good, it basically immediately says what user neeeds to know

**Comment by kof** [comment]
meaning title: 991px and down or 991px and up

**Comment by taylornowotny** [comment]
Makes sense, sorry I didn't specify.

For the title please use: "[min or max width value] and [direction]"

eg. "991px and down" = max-width 991px
eg. "1400px and up" = min-width 1400px

**Comment by TrySound** [comment]
Ah, ok. So same as before

**Comment by taylornowotny** [comment]
Thank you @TrySound !

**Pull request merged**